MiniSpace.APIGateway/src/MiniSpace.APIGateway/ntrada-async.yml

```diff
@@ -41,7 +41,7 @@ extensions:
       - Total-Count
 
   jwt:
-    issuerSigningKey: eiquief5phee9pazo0Faegaez9gohThailiur5woy2befiech1oarai4aiLi6ahVecah3ie9Aiz6Peij
+    issuerSigningKey: Gtn9vBDB5RCDLJSMqZQQmN75J8hgzbQwWkcD8jMIXnvCLAmlL0QVacUAbyootWihMrPIz
     validIssuer: minispace
     validateAudience: false
```

Comment:
Remember to handler signing key for production.

Reply:
Thank you, currently the production setting are protected. There will be a need to change it.